cmd/vic-machine/configure/configure.go

```diff
@@ -221,6 +231,14 @@ func (c *Configure) copyChangedConf(o *config.VirtualContainerHostConfigSpec, n
 	if n.RegistryCertificateAuthorities != nil {
 		o.RegistryCertificateAuthorities = n.RegistryCertificateAuthorities
 	}
+
+	if n.InsecureRegistries != nil {
```

Comment:
looks like you copied insecure registries and registry white list as well. If you plan to support those two configurations in this PR, you'll need a bit more work, since those two options are not certificate, so vic-machine inspect should display them correctly.
To make sure that work, you'll need to change code in this method https://github.com/vmware/vic/blob/master/lib/install/data/data.go#L278
And the registry white list get compiled if secure/insecure registry is configured at the same time during validation, so it'll be better to test different scenarios.